File: zircon.jvm.examples/src/main/java/org/hexworks/zircon/examples/other/WindowTitleExample.java
package org.hexworks.zircon.examples.other;

import org.hexworks.zircon.api.CP437TilesetResources;
import org.hexworks.zircon.api.SwingApplications;
import org.hexworks.zircon.api.application.AppConfig;
import org.hexworks.zircon.api.data.Size;
import org.hexworks.zircon.api.resource.TilesetResource;

public class WindowTitleExample {

    private static final Size GRID_SIZE = Size.create(20, 10);
    private static final TilesetResource TILESET = CP437TilesetResources.taffer20x20();

    public static void main(String[] args) {

        SwingApplications.startTileGrid(AppConfig.newBuilder()
                .withDefaultTileset(TILESET)
                .withSize(GRID_SIZE)
                .withTitle("Some cool title")
                .build());

    }

}

File: zircon.jvm.examples/src/main/kotlin/org/hexworks/zircon/examples/other/WindowTitleExample.kt
package org.hexworks.zircon.examples.other

import org.hexworks.zircon.api.CP437TilesetResources.taffer20x20
import org.hexworks.zircon.api.SwingApplications.startTileGrid
import org.hexworks.zircon.api.application.AppConfig.Companion.newBuilder
import org.hexworks.zircon.api.data.Size.Companion.create

object WindowTitleExample {

    private val GRID_SIZE = create(20, 10)
    private val TILESET = taffer20x20()

    @JvmStatic
    fun main(args: Array<String>) {
        startTileGrid(
            newBuilder()
                .withDefaultTileset(TILESET)
                .withSize(GRID_SIZE)
                .withTitle("Some cool title")
                .build()
        )
    }
}